packages/manager/modules/pci/src/projects/project/kubernetes/details/nodes/template.html

```diff
@@ -38,6 +45,11 @@
         </oui-column>
         <oui-action-menu data-disabled="$ctrl.Kubernetes.constructor.isProcessing($row.status) || $ctrl.Kubernetes.constructor.isProcessing($ctrl.cluster.status)" data-align="end" data-compact>
             <oui-action-menu-item data-text="{{:: 'kube_nodes_common_delete' | translate }}" data-on-click="$ctrl.deleteNode($row.id)"></oui-action-menu-item>
+            <oui-action-menu-item
+                data-ng-if="$row.billingType === 'hourly'"
```

Comment:
This should be handled elsewhere to avoid having to duplicate

Reply:
what duplicate? are you talking about constant 'hourly'?

Reply:
@radireddy yes that's what I had in mind, maybe `hourly` could be turned to a constant